pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -792,6 +837,9 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *promv1.Prome
 						Annotations: map[string]string{
 							"summary": "Some virt-handlers failed to roll out",
 						},
+						Labels: map[string]string{
+							"severity": "critical",
```

Comment:
@ashleyschuett  Hi, Is this the correct severity for this alert?

Reply:
@EdDev @ashleyschuett ?

Reply:
I'm not sure, I think it is, but i don't know what are threshold for critical is. Do we have any documentation or guide lines about what makes something critical or what the threshold is?

Reply:
@sradco 

I think we'd want to make this a `warning` and for it to be critical if zero handlers are present.